contracts/near/eth2-client/src/lib.rs

```diff
@@ -558,7 +570,11 @@ impl Eth2Client {
     /// Remove information about the headers that are at least as old as the given block number.
     fn gc_finalized_execution_blocks(&mut self, mut header_number: u64) {
         loop {
-            if self.finalized_execution_blocks.remove(&header_number).is_some() {
+            if self
+                .finalized_execution_blocks
+                .remove(&header_number)
+                .is_some()
+            {
```

Comment:
As the scope of this PR is quite narrow, let's not apply formatting fixes as this influences the binary.

```suggestion
            if self.finalized_execution_blocks.remove(&header_number).is_some() {
```